test/DynamoCoreTests/DSEvaluationViewModelTest.cs

```diff
@@ -115,8 +115,11 @@ public void AssertClassName(string guid, string className)
             string varname = GetVarName(guid);
             var mirror = GetRuntimeMirror(varname);
             Assert.IsNotNull(mirror);
```

Comment:
The RuntimeMirror check was alsways there, i guess it was to get the varname value